cmd/parca-agent/main.go

```diff
@@ -76,6 +77,17 @@ const (
 	// Use `sudo bpftool map` to determine the size of the maps.
 	defaultMemlockRLimit                   = 64 * 1024 * 1024  // ~64MB
 	defaultMemlockRLimitWithDWARFUnwinding = 512 * 1024 * 1024 // ~512MB
+
+	// We sample at 19Hz (19 times per second) because it is a prime number,
+	// and primes are good to avoid collisions with other things
+	// that may be happening periodically on a machine.
+	// In particular, 100 samples per second means every 10ms
+	// which is a frequency that may very well be used by user code,
+	// so a CPU profile could show a periodic workload on-CPU 100% of the time
+	// which is misleading.
+	defaultCPUSamplingFrequency = 19
+	// Setting the CPU sampling frequency too high can impact overall machine performance.
+	maxCPUSamplingFrequency = 150
```

Comment:
```suggestion
	maxAdvicedCPUSamplingFrequency = 150
```
Let's be super pedantic about the naming here.